schemas/key_link_reference.yaml

```diff
@@ -0,0 +1,36 @@
+key_link_reference:
```

Comment:
@BlumMichael should we rename this to just key_link? i don't see an advantage of having reference in the name and then it would also align with the path

Reply:
sure, will do rename. Thx!